CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 <!-- markdownlint-disable MD024 -->
 # Changelog
 
+### BREAKING CHANGES
+
+- [#17355](https://github.com/influxdata/telegraf/pull/17355): `inputs.opentelemetry` Changes the `profiles` support
+  from the `v1 experimental` to the `v1 development` as this experimental API is updated upstream. Will change
+  some metric fields, notably the removal of the `frame_type`, `stack_trace_id`, `build_id`, and `build_id_type` fields,
+  though other changes may also exist. Please refer to the OpenTelemetry documentation for more details:
+  https://opentelemetry.io/docs/
```

Comment:
Maybe
```suggestion
- PR [#17355](https://github.com/influxdata/telegraf/pull/17355) changes the `profiles` support
  of `inputs.opentelemetry` from the `v1 experimental` to the `v1 development` as this experimental API
  is updated upstream. This will change the metric by for example removing the no-longer reported
  `frame_type`, `stack_trace_id`, `build_id`, and `build_id_type` fields. Also the value format of other fields
  or tags might have changed. Please refer to the
  [OpenTelemetry documentation](https://opentelemetry.io/docs/)  for more details.
```